app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -115,6 +138,17 @@ def self.description
       @description ||= "Hawkular".freeze
     end
 
+    def self.event_monitor_class
+      ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher
+    end
+
+    # To blacklist defined event types by default, uncomment and add them here...
+    # def self.default_blacklisted_event_names
+    #  %w(
+    #  AddEventTypesHere
+    #  )
+    # end
+
```

Comment:
can you remove the comment and make it return an empty array instead?
imho that would be better documentation than a comment that can become stale